deployments/salt/splunk-otel-collector/collector_config.sls

```diff
@@ -26,11 +26,14 @@
 
 {% set splunk_listen_interface = salt['pillar.get']('splunk-otel-collector:splunk_listen_interface', '') %}
 
+{% set splunk_otel_collector_command_line_args = salt['pillar.get']('splunk-otel-collector:splunk_otel_collector_command_line_args', '') %}
+
 {% set collector_additional_env_vars = salt['pillar.get']('splunk-otel-collector:collector_additional_env_vars', {}) %}
 
 /etc/otel/collector/splunk-otel-collector.conf:
   file.managed:
     - contents: |
+        OTELCOL_OPTIONS={{ splunk_otel_collector_command_line_args }}
```

Comment:
The PR looks good!
Just a quick question as I am not sure if it's an issue, but do we need to sanitize the input especially that we are blindly setting it as env variable value?

Reply:
The scope of the env var is only the collector service, the onus is on the user to validate that they have valid command-line parameters, if not the collector service will fail at start time.